utils/TestDataGenerator.robot

```diff
@@ -2,12 +2,8 @@
 Library           String
 
 *** Keywords ***
-Get Random String
+Set Random String
```

Comment:
It is Get keywords. Setters are used when you want set property, value, pre-defigned by you